control/controlclient/direct.go

```diff
@@ -154,13 +155,18 @@ func NewDirect(opts Options) (*Direct, error) {
 			UseLastGood:      true,
 			LookupIPFallback: dnsfallback.Lookup,
 		}
-		dialer := netns.NewDialer()
+		var context func(context.Context, string, string) (net.Conn, error)
```

Comment:
I'd move all this stuff to netns itself, like we already have for other platforms:

```
bradfitz@tsdev:~/src/tailscale.com/net/netns$ git grep -i localhost
netns_darwin_tailscaled.go:             // Don't bind to an interface for localhost connections.
netns_windows.go:               // Don't bind to an interface for localhost connections,
```

Then this file is unchanged.

Reply:
Done.